release-notes/README.md

```diff
@@ -191,6 +210,19 @@ Please note that this release introduces breaking changes. For more details, see
 
 This SDK is compatible with Android Core 1.8.2 and above.
 
+## June 9, 2021
+
+### Flutter
+
+- Upgrade to Flutter 2 and Null-safety support for following packages:
```

Comment:
Same here:
"Upgrade plugin to support Flutter 2 and null safety support." (with full decimal version for Flutter)